CMakeLists.txt

```diff
@@ -367,27 +367,37 @@ if(WITH_CUDA)
   include("${PCL_SOURCE_DIR}/cmake/pcl_find_cuda.cmake")
 endif()
 
+
+# Find VTK
+option(WITH_VTK "Build VTK-Visualizations" TRUE)
+if(WITH_VTK AND NOT ANDROID)
+  include("${PCL_SOURCE_DIR}/cmake/pcl_find_vtk.cmake")
+else()
+  set(VTK_FOUND OFF)
+endif()
+
+#VTK can depend on Qt and search for its required version, so search after so we can overwrite Qt5_FOUND if the version we require is not found.
 option(WITH_QT "Build QT Front-End" TRUE)
 if(WITH_QT)
-  find_package(Qt5 COMPONENTS Concurrent OpenGL Widgets QUIET)
+  find_package(Qt5 5.9.5 COMPONENTS Concurrent OpenGL Widgets)
   if(Qt5_FOUND)
+    message(STATUS "Qt5 version: ${Qt5_VERSION}")
     #Set Cmake Auto features to skip .hh files
     if(POLICY CMP0100)
       cmake_policy(SET CMP0100 OLD)
     endif()
-    message(STATUS "Found Qt5 version: ${Qt5_VERSION}")
-  endif()
-endif()
+    
+    get_property(core_def TARGET Qt5::Core PROPERTY INTERFACE_COMPILE_DEFINITIONS)
+    list(APPEND core_def "QT_DISABLE_DEPRECATED_BEFORE=0x050900")
```

Comment:
Perhaps move the string to a variable created around L382 for better visibility when bumping the versions in future